src/antsibull_changelog/cli.py

```diff
@@ -373,6 +373,7 @@ def _get_refresh_config(args: Any,
             elif part == 'fragments-without-archives':
                 refresh_fragments = 'without-archives'
             else:
+                # pylint:disable-next=broad-exception-raised
                 raise Exception(
```

Comment:
This should raise a ValueError

Reply:
Yes, but that's not a change that should happen in this PR. It potentially modifies behavior for calling code.

Reply:
try: ... except Exception: will work just the same, but perhaps it would be better suited for a new PR.